addons/hitreactions/XEH_postInit.sqf

```diff
@@ -0,0 +1,65 @@
+#include "script_component.hpp"
+
+GVAR(armComponents) = [
+    "leftshoulder",
+    "rightshoulder",
+    "lefthand",
+    "leftforearm",
+    "leftarmroll",
+    "rightforearm",
+    "rightarmroll",
+    "righthand",
+    "rightarm"
+];
+
+GVAR(safePickupDistance) = DEFAULT_PICKUP_DISTANCE;
+if (isNil QGVAR(undroppableGuns)) then {GVAR(undroppableGuns) = DEFAULT_UNDROPPABLE_GUNS};
```

Comment:
How would it not be `nil`?

This and everything above fits `preInit` more.

Reply:
user-defined in before postInit

Reply:
@LinkIsGrim just move that and everything above to preinit and skip isnil check?

Reply:
Moving to preInit is fine, isNil check should be kept. What if user's addon loaded first and defined that in preInit as well?